extmod/modssl_mbedtls.c

```diff
@@ -300,6 +540,12 @@ STATIC const mp_rom_map_elem_t ssl_context_locals_dict_table[] = {
     #if MICROPY_PY_SSL_FINALISER
     { MP_ROM_QSTR(MP_QSTR___del__), MP_ROM_PTR(&ssl_context___del___obj) },
     #endif
+    #ifdef MICROPY_SSL_MBEDTLS_EXTRAS
```

Comment:
I don't think we need this config option.  We want to migrate all users of this ssl module to use these new methods to load certificates, so they should be unconditionally available.

Reply:
I added this because not every board (i.e. in stm32 port) could fit the extra bytes and the build process failed...
So this seemed the best option, to be able to enable these features at port or board level...

Reply:
I've removed the macro and it seems the build process is OK in stm32 port now 👍🏼